packages/marko-web-identity-x/browser/form/fields/address-extra.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <form-group class-name="col-md-4">
+  <form-group :class-name="className">
```

Comment:
Can you verify that this prop is already present in all of the other components you added it to in `custom-column.vue`? If not, it should be added.

Reply:
@solocommand  this has been added to the field components that where missing it.